samples/uss-profile-sample/package.json

```diff
@@ -33,7 +33,7 @@
     "watch": "tsc -watch -p ./"
   },
   "dependencies": {
-    "@zowe/cli": "8.0.0-next.202402132108",
+    "@zowe/cli": "8.0.0-next.202402261705",
```

Comment:
I believe we will be doing version ranges going forward (at least for samples) right?
cc: @t1m0thyj

Reply:
I think it would be convenient for us to use version ranges going forward 😋 No need to change anything in this PR though, we can do that when migrating to the SDKs in https://github.com/zowe/vscode-extension-for-zowe/pull/2730